addons/payment_adyen/controllers/main.py

```diff
@@ -245,6 +245,10 @@ def adyen_notification(self):
                 acquirer_sudo = PaymentTransaction.sudo()._get_tx_from_feedback_data(
                     'adyen', notification_data
                 ).acquirer_id  # Find the acquirer based on the transaction
+            except ValidationError:
+                # Avoid noise in the logs when the (pos, payment)_adyen are both installed
```

Comment:
Sorry, I forgot to submit that one :sweat_smile: 
I believe it better explains why the try/except has been split in two.
```suggestion
                # Warn rather than log the traceback to avoid noise when a POS payment notification
                # is received and the corresponding `payment.transaction` record is not found.
```